### PR TITLE
Weaviate guide - code and examples

### PR DESCRIPTION
**Pull request by ganochenkodg**
TF code and yaml manifests for Weaviate

### CODE REVIEW

**Comment by bourgeoisor** [comment]
@ganochenkodg this is for an upcoming GKE tutorial, correct?
ping me when the PR is ready for review

**Comment by ganochenkodg** [comment]
> @ganochenkodg this is for an upcoming GKE tutorial, correct? ping me when the PR is ready for review

Yes, for upcoming guide, review it pls

File: databases/weaviate/docker/chatbot/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM python:3.12-slim-bookworm
```
Comment (bourgeoisor) [inline databases/weaviate/docker/chatbot/Dockerfile:1]: Could you create a new workflow for this weaviate samples, that has 4 steps:
- 2x `docker build` steps for the two containers
- 2x `terraform validate` steps for the two terraform directories

You can find an example here: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/workflows/cost-optimization-gke-vpa-recommendations-ci.yml

Comment (ganochenkodg) [reply databases/weaviate/docker/chatbot/Dockerfile:1]: yep, sure, will add

Comment (ganochenkodg) [reply databases/weaviate/docker/chatbot/Dockerfile:1]: done

**Review by bourgeoisor — APPROVED** [review]
LGTM

**Pull request merged**